pyHalo/Halos/HaloModels/ULDM.py

```diff
@@ -52,26 +54,17 @@ def lenstronomy_params(self):
         """
         See documentation in base class (Halos/halo_base.py)
         """
-
-        # Copied from the TNFW class
-        [concentration, rt, theta_c, kappa_0] = self.profile_args
-        Rs_angle, theta_Rs = self._lens_cosmo.nfw_physical2angle(self.mass, concentration, self.z)
-
+        
+        [concentration, theta_c, kappa_0] = self.profile_args
         x, y = np.round(self.x, 4), np.round(self.y, 4)
-
-        Rs_angle = np.round(Rs_angle, 10)
-        theta_Rs = np.round(theta_Rs, 10)
-        r_trunc_arcsec = rt / self._lens_cosmo.cosmo.kpc_proper_per_asec(self.z)
-
-        kwargs_nfw_temporary = {'alpha_Rs': theta_Rs, 'Rs': Rs_angle,
-                  'center_x': x, 'center_y': y, 'r_trunc': r_trunc_arcsec}
-
-        # need to specify the keyword arguments for the ULDM profile and renormalize NFW profile
-        kwargs_uldm = {'theta_c': theta_c, 'kappa_0': kappa_0,
-                'center_x': x, 'center_y': y}
-        kwargs_nfw = self._rescaled_tnfw_params(kwargs_nfw_temporary, kwargs_uldm)
-        kwargs = [kwargs_nfw, kwargs_uldm]
-
+        Rs_angle, alpha_Rs = self._lens_cosmo.nfw_physical2angle(self.mass, concentration, self.z)
```

Comment:
did we discuss adding a cnfw_physical2angle function here? otherwise the normalization might be a tad off